PennMobile/Events/EventsTableViewCell.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  EventsTableViewCell.swift
+//  PennMobile
+//
+//  Created by Samantha Su on 10/1/21.
+//  Copyright © 2021 PennLabs. All rights reserved.
+//
+
+import UIKit
+import Kingfisher
+import SwiftSoup
+
+class EventsTableViewCell: UITableViewCell {
```

Comment:
Following the naming convention of `PennEvents` and also because we'll likely incorporate other type of events in the future (e.g. ClubEvents), lets name the view controller as PennEventsTableViewCell()